_includes/pbc/all/install-features/202410.0/install-glue-api/install-the-payments-glue-api.md

```diff
@@ -173,12 +174,100 @@ class GlueApplicationDependencyProvider extends SprykerGlueApplicationDependency
 
         return $resourceRelationshipCollection;
     }
+    
+     /**
+     * {@inheritDoc}
+     *
+     * @return array<\Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ResourceRoutePluginInterface>
+     */
+    protected function getResourceRoutePlugins(): array
+    {
+        return [
+            new PaymentCustomersResourceRoutePlugin(),
+        ];
+    }
 }
 ```
 
 {% info_block warningBox "Verification" %}
 
 To verify `PaymentMethodsByCheckoutDataResourceRelationshipPlugin` is activated, send a POST request to `https://glue.mysprykershop.com/checkout-data?include=payment-methods` and make sure that `checkout-data` resource has a relationship to the `payment-methods` resources.
+To verify `PaymentCustomersResourceRoutePlugin` is activated, send a POST request to `https://glue.mysprykershop.com/payment-customers` and make sure that you get a response with customer data.
+
+An example request for `https://glue.mysprykershop.com/payment-customers`:
+The `customerPaymentServiceProviderData` field is a placeholder for the data that should be returned by the PSP widget.
+
+```json
+{
+  "data": {
+    "type": "payment-customers",
+    "attributes": {
+      "payment": {
+        "paymentMethodName": "dummyPayment",
+        "paymentProviderName": "dummyPaymentProvider"
+      },
+      "customerPaymentServiceProviderData": "customerPaymentServiceProviderData"
+    }
+  }
+}
+```
+
+Depending on the payment method, the response may vary.
```

Comment:
from the description it's not clear what use case is covered. Is this customer the current logged-in customer, or who is the customer?

(I know that there `customerPaymentServiceProviderData` should have token or some id from payment provider, but it's not clear).

I suggest improve the description and write use-case and update `customerPaymentServiceProviderData` to be closer to real use-case